File: Modul 6/Bahasa Python (modul 6)/PRAK601-2310817320015-AlfisyahRinaFajriati.py
baris, kolom = map(int,input().split()) 

matrix = [[0] * kolom for _ in range(baris)]

input_flattened = list(map(int,input().split()))
index = 0
for i in range(baris):
    for j in range(kolom):
        matrix[i][j] = input_flattened[index]
        index += 1

for i in range(baris):
    for j in range(kolom):
        print(f"{matrix[i][j]:3}", end="")
    print()

File: Modul 6/Bahasa C (modul 6)/PRAK601-2310817320015-AlfisyahRinaFajriati.c
#include <stdio.h>
int main(){

    int baris, kolom;

    printf("");
    scanf("%d", &baris);
    printf("");
    scanf("%d", &kolom);

    int matrix[baris][kolom];
    int i, j;

    for (i = 0; i < baris; i++) {
        for (j = 0; j < kolom; j++) {
            printf("", i + 1, j + 1);
            scanf("%d", &matrix[i][j]);
        }
    }

    for (i = 0; i < baris; i++) {
        for (j = 0; j < kolom; j++) {
            printf("%3d", matrix[i][j]);
        }
    printf("\n");
    }

    return 0;
}